jobs/config.json

```diff
@@ -9918,6 +9918,22 @@
       "sig-scalability"
     ]
   },
+  "ci-kubernetes-local-e2e": {
+    "args": [
+      "--up",
+      "--provider=local",
+      "--deployment=local",
+      "--test",
+      "--extract=ci/latest",
+      "--ginkgo-parallel=1",
+      "--test_args=--ginkgo.focus=\\[Feature:Conformance\\]",
+      "--timeout=320m"
```

Comment:
I don't think your conformance test should take 320m?

Reply:
dropped it down to 120 :)